.github/workflows/cli-verification.yaml

```diff
@@ -99,13 +99,20 @@ jobs:
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
               run: |
-                  npm init -y
-                  npm install nango@npm:@nangohq/cli@0.0.1-$GIT_HASH
+                  # Install the cli from the github package registry
+                  npm install -g nango@npm:@nangohq/cli@0.0.1-$GIT_HASH
```

Comment:
not familiar with the syntax. What part mean the package is fetched from github package registry?